src/libxively/xi_config.h

```diff
@@ -32,12 +32,8 @@
 #endif
 
 #ifndef XI_MQTT_PORT
-#ifdef XI_DEBUG_NO_TLS
-#define XI_MQTT_PORT 1883
-#else
 #define XI_MQTT_PORT 8883
```

Comment:
Perhaps leave a comment here that port 1883 can be used for insecure connections.